### DISCUSSION TITLE
Bottom tabs rounded corners with background color not work

### DISCUSSION THREAD

**Discussion by tastafur** [react-navigation/react-navigation#10280] [Q&A] [answered]
### Current behavior

There is no way to remove the background of the tab navigator bottom at least in android, I have tried to make it transparent by stepping on the parent color of the theme and react native complains saying that it uses it and I cannot change it

![Screenshot_20220121-103458_DoGood](https://user-images.githubusercontent.com/11853181/150505001-a44b8ec4-3119-45f6-ad64-db065a91741c.jpg)

### Expected behavior

In the photo that I have shared before there is a part in the bottom tab that shows red, in some way it can be transparent and thus the bottom tab can be rounded

### Reproduction

I have not

### Platform

- [X] Android
- [ ] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [X] @react-navigation/bottom-tabs
- [X] @react-navigation/stack
- [X] @react-navigation/native-stack

### Environment

- [X] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               |  6.0.6
| @react-navigation/bottom-tabs          | 6.0.9
| @react-navigation/stack                | 6.0.11
| react-native-safe-area-context         | 3.3.2
| react-native-screens                   | 3.9.0
| react-native-gesture-handler           | 1.10.3
| react-native-reanimated                | 1.13.2
| react-native-tab-view                  | 3.1.1
| react-native                           | 0.66.3
| node                                   | 16.10.0
| yarn                            | 1.22.0

**Comment by asadbjs** [comment, marked as answer]
**Thnx me later...**
tabBarOptions={{
        style: {
          backgroundColor: 'red',
          borderTopLeftRadius: 25,
          borderTopRightRadius: 25,
          position: "absolute",
          bottom: 0,
        },
      }}

**Reply by blazk0** [reply]
Just beware that this will be visible over content in scrollview so you have to add paddingBottom manually to each scrollView

**Reply by Sxm02** [reply]
Thanks to you man.
My problme was because of not using position:"absoolute"
Appreciate it

**Reply by asadbjs** [reply]
@Sxm02 Most Welcome

**Reply by mosesjohn6969** [reply]
solved my issue thanks man

**Reply by Ahsanali012** [reply]
Saved the day...

**Reply by pedroagcf** [reply]
solved my issue! thanks

**Reply by durejagaurav** [reply]
But with position absolute, scrollviews item going behind the navigator, not scroll till the top of bottom navigator

**Reply by kingdevil731** [reply]
exactly @durejagaurav, this solution fixes one problem but introduces another one

**Reply by salahbm** [reply]
Could you solve this issue?

**Reply by sheharyar2** [reply]
not working in my case ,
  screenOptions={{
           headerShown: false,
          tabBarHideOnKeyboard:true,
          tabBarStyle: {
            
            position: 'absolute'
          },
        }}

**Reply by FareeKhan** [reply]
this i not working

**Comment by salahbm** [comment]
Settings position:'absolute', doesnt solve the issue when I use ScrollView in the same page with tabs. Any ideas? Thanks ~~

**Reply by mohamedyasser27** [reply]
+1

**Comment by Eumentis-Jayashree** [comment]
> ### Current behavior
> There is no way to remove the background of the tab navigator bottom, at least in android, I have tried to make it transparent by stepping on the parent color of the theme, but react native complains that it uses it and I cannot change it
> 
> ![Screenshot_20220121-103458_DoGood](https://user-images.githubusercontent.com/11853181/150505001-a44b8ec4-3119-45f6-ad64-db065a91741c.jpg)
> 
> ### Expected behavior
In the photo that I have shared before, there is a part in the bottom tab that shows red, In some way, it can be transparent and thus the bottom tab can be rounded
> 
> ### Reproduction
I have not
> 
> ### Platform
> * [x]  Android
> * [ ]  iOS
> * [ ]  Web
> * [ ]  Windows
> * [ ]  MacOS
> 
> ### Packages
> * [x]  @react-navigation/bottom-tabs
> * [x]  @react-navigation/stack
> * [x]  @react-navigation/native-stack
> 
> ### Environment
> * [x]  I've removed the packages that I don't use
> 
> package	version
> @react-navigation/native	6.0.6
> @react-navigation/bottom-tabs	6.0.9
> @react-navigation/stack	6.0.11
> react-native-safe-area-context	3.3.2
> react-native-screens	3.9.0
> react-native-gesture-handler	1.10.3
> react-native-reanimated	1.13.2
> react-native-tab-view	3.1.1
> react-native	0.66.3
> node	16.10.0
> yarn	1.22.0

I tried this solution and it's working:
1. Used this style: 
 ````
```
tabBarStyle: {
        backgroundColor: 'white',
        borderTopEndRadius: 20,
        borderTopStartRadius: 20,
      }
```
````
3. Updated react native theme background color to current screen's background color. which helps to get rid of that side corners (in your case those are in red color).
 ````
```
<NavigationContainer
              theme={{
                ...DefaultTheme,
                colors: { ...DefaultTheme.colors, background: "#F8F8F8"},
              }}
```
````
 Here "#F8F8F8" is my screen's background color.
